src/tls13.c

```diff
@@ -1371,9 +1371,14 @@ int DeriveTls13Keys(WOLFSSL* ssl, int secret, int side, int store)
     word32 TimeNowInMilliseconds(void)
     {
         struct timeval now;
-
+#ifdef FUSION_RTOS
+        if (FCL_GETTIMEOFDAY(&now, 0) < 0)
+            return (word32)GETTIME_ERROR;
+            //TODO research why negative error code returns as word32
```

Comment:
Can we resolve this TODO?

Reply:
Evaluated but I don't see a way to resolve, time wants to be non-negative so an unsigned return type is the only option for time calls across the spectrum perhaps we could consider returning simply WOLFSSL_FAILURE (IE zero) here instead of a negative return code that gets cast to an unsigned value.

Reply:
Attempted to return WOLFSSL_FAILURE but resulted in tests breaking and would likely effect user applications in use today. We'll need to revisit this in a stand-alone PR outside this porting effort. Did refactor this section however removing porting changes from the default and adding to a stand-alone section for the OS.